packages/create-content-sdk-app/src/templates/nextjs-app-router (beta)/src/i18n/routing.ts

```diff
@@ -0,0 +1,12 @@
+import { defineRouting } from 'next-intl/routing';
+
+export const routing = defineRouting({
+  // A list of all locales that are supported
+  locales: ['en', 'de-DE'],
```

Comment:
Can the default language be taken from `SitecoreConfig`?
```suggestion
  locales: [config.defaultLanguage],
```